3.0/s2i/bin/assemble

```diff
@@ -8,8 +8,7 @@ function rake_assets_precompile() {
   ! bundle exec 'rake -T' | grep "assets:precompile" >/dev/null && return
 
   echo "---> Starting asset compilation ..."
-  bundle exec "rake assets:precompile"
-}
+  bundle exec rake assets:precompile
```

Comment:
Please leave the quotes, bundler fails without them because of some internal command optimization.
```suggestion
  bundle exec "rake assets:precompile"
```

The bundler error if this is unquoted:
```
---> Starting asset compilation ...
rake aborted!
LoadError: cannot load such file -- bundler/setup
/opt/app-root/src/config/boot.rb:3:in `require'
/opt/app-root/src/config/boot.rb:3:in `<top (required)>'
/opt/app-root/src/config/application.rb:1:in `require_relative'
/opt/app-root/src/config/application.rb:1:in `<top (required)>'
/opt/app-root/src/Rakefile:4:in `require_relative'
/opt/app-root/src/Rakefile:4:in `<top (required)>'
/opt/app-root/src/bundle/ruby/3.0.0/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `load'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:63:in `kernel_load'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli/exec.rb:28:in `run'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli.rb:474:in `exec'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli.rb:30:in `dispatch'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/cli.rb:24:in `start'
/usr/share/gems/gems/bundler-2.2.22/libexec/bundle:49:in `block in <top (required)>'
/usr/share/gems/gems/bundler-2.2.22/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
/usr/share/gems/gems/bundler-2.2.22/libexec/bundle:37:in `<top (required)>'
/opt/app-root/src/bin/bundle:3:in `load'
/opt/app-root/src/bin/bundle:3:in `<main>'
```